cv32e40s.manual_expression_cov_todo.vRefine

```diff
@@ -0,0 +1,93 @@
+    <rule cb-checksum="238500707" ccType="type" ccfFlagsMask="4398046513408-42" comment="27" dcExpr="dummy_insert ? dummy_instr : (seq_valid ? seq_instr : instr_decompressed)" dcExprIndex="0" domain="icc" entityName="cv32e40s_if_stage/15/1/4" entityType="min-term" excTime="1693224032" file="28" im-checksum="32998194" line="544" name="exclude" packageName="worklib" reviewer="unknown" text="dummy_insert ? dummy_instr : (seq_valid ? seq_instr : instr_decompressed)" ung="0" user="16" vscope="default"></rule>
+    <rule cb-checksum="238500707" ccType="type" ccfFlagsMask="4398046513408-42" comment="27" dcExpr="dummy_insert ? dummy_instr : (seq_valid ? seq_instr : instr_decompressed)" dcExprIndex="0" domain="icc" entityName="cv32e40s_if_stage/15/1/5" entityType="min-term" excTime="1693224032" file="28" im-checksum="32998194" line="544" name="exclude" packageName="worklib" reviewer="unknown" text="dummy_insert ? dummy_instr : (seq_valid ? seq_instr : instr_decompressed)" ung="0" user="16" vscope="default"></rule>
+    <rule cb-checksum="238500707" ccType="type" ccfFlagsMask="4398046513408-42" comment="27" dcExpr="dummy_insert ? dummy_instr : (seq_valid ? seq_instr : instr_decompressed)" dcExprIndex="0" domain="icc" entityName="cv32e40s_if_stage/15/1/6" entityType="min-term" excTime="1693224032" file="28" im-checksum="32998194" line="544" name="exclude" packageName="worklib" reviewer="unknown" text="dummy_insert ? dummy_instr : (seq_valid ? seq_instr : instr_decompressed)" ung="0" user="16" vscope="default"></rule>
+    <rule cb-checksum="2103009422" ccType="type" ccfFlagsMask="4398046513408-42" comment="14" dcExpr="(dummy_instr_id_i || hint_instr_id_i) ? mem[0] : RF_REG_RV" dcExprIndex="0" domain="icc" entityName="&quot;cv32e40s_register_file.genblk1_T&quot;/1/1/5" entityType="min-term" excTime="1693230836" file="29" im-checksum="1144468874" line="73" name="exclude" packageName="worklib" reviewer="unknown" text="(dummy_instr_id_i || hint_instr_id_i) ? mem[0] : RF_REG_RV" ung="0" user="16" vscope="default"></rule>
+    <rule cb-checksum="2103009422" ccType="type" ccfFlagsMask="4398046513408-42" comment="14" dcExpr="(dummy_instr_id_i || hint_instr_id_i) ? mem[0] : RF_REG_RV" dcExprIndex="0" domain="icc" entityName="&quot;cv32e40s_register_file.genblk1_T&quot;/1/1/6" entityType="min-term" excTime="1693230836" file="29" im-checksum="1144468874" line="73" name="exclude" packageName="worklib" reviewer="unknown" text="(dummy_instr_id_i || hint_instr_id_i) ? mem[0] : RF_REG_RV" ung="0" user="16" vscope="default"></rule>
+  </rules>
+  <cache-map>
+    <cache-entry key="0" value="Glitch controll signal. There is no test using glitches."></cache-entry>
```

Comment:
_"There is no test using glitches"_.
Is this exclusion placed in a "todo" refinement file because we should evaluate later whether we want to add simulation tests using glitches?

Reply:
Yess. I don't know if it is necessary because it should be covered in formal, but yea, we should evaluate it.